.github/actions/docker/main_docker.sh

```diff
@@ -0,0 +1,14 @@
+#! /bin/sh
+# Require a valid Git ref as an argument
+if [ "$#" -ne 1 ] || ! git rev-parse --verify "$1" >/dev/null; then
+    echo "Usage: $0 GIT_REF TAG_NAME" >&2
```

Comment:
These arguments aren't actually getting used, are they?

Reply:
I am not 100% sure what this does.  I got the impression it verified that this was actually part of the git repo to ensure that the right structure was in place.    I copied it from the buildenv repo